transients_nearby_galaxies/transients_nearby_galaxies.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
```

Comment:
<p>lk.interact() will be available during the workshop and the current plan is for our patch to be integrated into the next version of TIKE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>Should I rewrite this portion of the tutorial using lk.interact() or just keep it as it is and remove this reference to it?</p>

Reply:
<p>I don't think there's any advantage to using the interact feature, since this relies so much on the background subtraction. So I'll just remove the reference to it. </p>

Reply:
<p>This is also a good idea. I like this method of "getting our hands dirty" without the help of lightkurve</p><p> </p>